playbooks/common/openshift-cluster/upgrades/upgrade_control_plane.yml

```diff
@@ -102,12 +112,26 @@
       state: link
     when: ca_crt_stat.stat.isreg and not ca_bundle_stat.stat.exists
 
+  # Run the upgrade hook prior to restarting services/system if defined:
+  - debug: msg="Running upgrade hook {{ openshift_master_upgrade_hook }}"
```

Comment:
@ashcrow The variable interpolation wouldn't work in the task name. I'm good with having the debugs in place to ensure that we output the hook location in the output.